proxy/envoy/policy.go

```diff
@@ -61,6 +61,7 @@ func insertMixerFilter(listeners []*Listener, instances []*model.ServiceInstance
 							"source.ip":  context.IPAddress,
 							"source.uid": context.UID,
 						},
+						QuotaName: "RequestCount",
```

Comment:
I was wondering about that. I thought that this causes the mixer to get called for all requests, which is a performance issue, but I'm assuming mixer would only rate limit the calls to the ratings service, since the mixer config has a `target.service` selector. Am I wrong?
```
    - selector: target.service == "ratings.*"
      aspects:
      - kind: quotas
        params:
          quotas:
          - descriptor_name: RequestCount
```